src/main/scala/nl/knaw/dans/easy/stage/fileitem/EasyStageFileItem.scala

```diff
@@ -19,16 +19,17 @@ import java.io.File
 import java.sql.SQLException
 
 import com.yourmediashelf.fedora.client.FedoraClientException
-import nl.knaw.dans.easy.stage.lib.FOXML.{getDirFOXML, getFileFOXML}
+import nl.knaw.dans.easy.stage.lib.FOXML.{ getDirFOXML, getFileFOXML }
 import nl.knaw.dans.easy.stage.lib.Util._
 import nl.knaw.dans.easy.stage.lib._
+import nl.knaw.dans.easy.stage._
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
 import org.apache.commons.configuration.PropertiesConfiguration
-import org.slf4j.{Logger, LoggerFactory}
 
-import scala.util.{Failure, Success, Try}
+import scala.annotation.tailrec
+import scala.util.{ Failure, Success, Try }
 
-object EasyStageFileItem {
-  val log: Logger = LoggerFactory.getLogger(getClass)
+object EasyStageFileItem extends DebugEnhancedLogging {
```

Comment:
Funny, you exchange our own logger for `DebugEnhancedLogging` 👍  but you still use the `log` from `nl.knaw.dans.easy.stage.lib.Util`...